### ISSUE TITLE
How to Join link

### ISSUE THREAD

**Issue by JohnLZeller** [OSURoboticsClub/Website#37]

**Comment by Nrpickle** [comment]
Lol, sorry guys. Wasn't 100% sure how that works.

**Comment by Gzxgzxgzx** [comment]
I think the "How to join" section on the main page covers this?

**Comment by Nrpickle** [comment]
It does, but I want Dylan to write a much more indepth version that covers how to actually get involved in the club, instead of just actually join. Links to mailing lists for the main teams, etc.

If you could bug him about this too, that would be awesome.

**Comment by Gzxgzxgzx** [comment]
Alright. I will commence with the bugging of Dylan.

**Comment by thrushd** [comment]
yeah yeah.... I'll get it to you by tomorrow.

On Tue, Feb 16, 2016 at 9:00 PM Harlan Waldrop <notifications@github.com>
wrote:

> Alright. I will commence with the bugging of Dylan.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OSURoboticsClub/Website/issues/37#issuecomment-185024402>
> .
>
-- 
Dylan Thrush
Vice President | OSU Robotics Club
<http://groups.engr.oregonstate.edu/osurc/>
Mechanical Automation Engineer | SARL <http://ehsc.oregonstate.edu/sarl>
Lab Manager | mLab <http://research.engr.oregonstate.edu/mlab/>
Website | dylanthrush.com
Email | dylanthrush1@gmail.com
Phone | (425) 314-4753 (Call / text anytime)

**Comment by thrushd** [comment]
Ok here you go:

"Joining the Robotics Club is really easy! First thing you should do is
join the general club mailing list, located here
<https://secure.engr.oregonstate.edu/mailman/listinfo/osurcmembers>. To
actually become an official member you will need to stop by our lab (Graf
206
<https://www.google.com/maps/place/Graf+Hall,+1680+SW+Monroe+Ave,+Corvallis,+OR+97331/@44.5676676,-123.2770557,17z/data=!3m1!4b1!4m2!3m1!1s0x54c040bdc4461ea9:0x981a3b57c9919466>)
during officer hours (which can be found on our calendar here
<http://osurobotics.club/calendar/>, which is also posted outside the lab).
An officer will have you fill out a short form and then process the fee
($10 for Fall, $8 for Winter, $5 for Spring), membership lasts from when
you pay to the beginning of fall term next school year, hence the price
break for joining later in the year. Membership will give you access to all
of the clubs resources, which includes computers, electronics tools, hand
tools and a ton of other useful stuff for building robots! You can use the
lab as you please at any time (you will get a door code).
If you are a bit more motivated you can join one of our project teams,
whose contact information is listed below:

Mars Rover Team
Team Lead: Billy Edwards (edwardwi@oregonstate.edu)

Arial Team
Team Lead: Evan Gonnerman (gonnerme@oregonstate.edu)

Underwater Team
Team Lead: Dan Knott (knottd@oregonstate.edu)

Hope to see you soon!
*make more robot"*

Something like that, feel free to edit, also to format the contact info of
team leads. Did I miss anything?

On Tue, Feb 16, 2016 at 10:38 PM Dylan Thrush <dylanthrush1@gmail.com>
wrote:

> yeah yeah.... I'll get it to you by tomorrow.
>
> On Tue, Feb 16, 2016 at 9:00 PM Harlan Waldrop <notifications@github.com>
> wrote:
>
>> Alright. I will commence with the bugging of Dylan.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/OSURoboticsClub/Website/issues/37#issuecomment-185024402>
>> .
>>
> --
> Dylan Thrush
> Vice President | OSU Robotics Club
> <http://groups.engr.oregonstate.edu/osurc/>
> Mechanical Automation Engineer | SARL <http://ehsc.oregonstate.edu/sarl>
> Lab Manager | mLab <http://research.engr.oregonstate.edu/mlab/>
> Website | dylanthrush.com
> Email | dylanthrush1@gmail.com
> Phone | (425) 314-4753 (Call / text anytime)
>
-- 
Dylan Thrush
Vice President | OSU Robotics Club
<http://groups.engr.oregonstate.edu/osurc/>
Mechanical Automation Engineer | SARL <http://ehsc.oregonstate.edu/sarl>
Lab Manager | mLab <http://research.engr.oregonstate.edu/mlab/>
Website | dylanthrush.com
Email | dylanthrush1@gmail.com
Phone | (425) 314-4753 (Call / text anytime)

**Comment by Gzxgzxgzx** [comment]
Index updated to reflect these changes: http://osurobotics.club/

The information covered looks solid to me.

**Comment by Nrpickle** [comment]
Yeah, looks good, but we need to make a page dedicated to our lab space.

Also, maybe mention email "our Vice President, Dylan Thrush"?
